docs/implementing-tests.md

```diff
@@ -0,0 +1,72 @@
+# Implementing Tests in Coalton
+
+Letting Coalton run loose in a Fiasco test suite can cause a bit of havoc, but, with a little finesse, Coalton and Fiasco can interact safely and smoothly.
+
+## An Example Coalton Package:
+```
+(defpackage #:foobar
+  (:use #:coalton
+        #:coalton-prelude)
+  (:export #:add-twenty
+           #:baz-it))
+
+(in-package #:foobar)
+
+(coalton-toplevel
+  (define (add-twenty x)
+    (+ x 20))
+    
+  (define (baz-it string)
+    (coalton-library/string:concat "baz-" string)))
+```
+## Defining the test package
+
+When defining a test package for a Coalton project, you'll actually make two separate packages, one for native Coalton tests, and another that will alias your Coalton tests into a Fiasco test package (in addition to any Common Lisp specific tests you want to implement).
+
+### First you will define the Fiasco test package for your project:
```

Comment:
This is a little bikesheddy but could you not put verbiage like this in section headings (here and elsewhere), and instead make it paragraph content? Maybe something like:

```
### Step 1: Define a package

First you will define ...
```